packages/cpt-ui/__tests__/BasicDetailsSearchResultsPage.test.tsx

```diff
@@ -0,0 +1,165 @@
+import React from "react"
+import {render, screen, fireEvent} from "@testing-library/react"
+import {MemoryRouter} from "react-router-dom"
+import BasicDetailsSearchResultsPage from "@/pages/BasicDetailsSearchResultsPage"
+import {SearchResultsPageStrings} from "@/constants/ui-strings/BasicDetailsSearchResultsPageStrings"
+import {FRONTEND_PATHS} from "@/constants/environment"
+
+const mockNavigate = jest.fn()
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useNavigate: () => mockNavigate
+}))
+
+// TODO: update tests when the actual data is queriable so we aren't testing against hardcoded values.
```

Comment:
If we know the form that the response is going to take already (I think we do?) then there's no reason not to set up a jest mock for `axios` or `fetch` which returns that data now.